src/home/fetch-github/fetch-issues-preview.ts

```diff
@@ -29,7 +29,7 @@ async function checkPrivateRepoAccess(): Promise<boolean> {
         return false;
       } else {
         // Handle other errors if needed
-        console.error("Error checking repository access:", error);
+        showError(`${error}`, false, "Error checking repository access:");
```

Comment:
Should it be
```TypeScript
showError("Error checking repository access:", false, `${error}`);
```

Since error could potentially be empty? Or is it fine that way